src/how_to/propose.md

```diff
@@ -1,3 +1,129 @@
+
+LangTeamChampion["Are you an experienced contributor\nwith a lang team champion?"]
+
+subgraph Stages
+    ExperimentalFeatureGateProposed["Create a tracking issue and\nopen a rust-lang/rust PR proposing\nan experimental feature gate"]
+    ExperimentalFeatureGateAccepted["Experiment is approved. Write code!"]
+    RFCOpen["Open a rust-lang/rfcs PR\nwith the completed RFC"]
+    RFCAccepted["RFC is merged.\nIf needed, create a tracking issue,\n and finalize the implementation"]
+    FeatureComplete["Implementation is complete.\nGain experience with the new feature."]
+    StabilizationProposed["Author a stabilization report and\nopen a PR stabilizing the feature"]
+    Documented["Author a rust-lang/reference PR\nextending the reference\nto describe the new feature."]
+    StabilizationAccepted["Stabilization is approved!\nFeature rides the trains into stable."]
+    ChangeProposed["Open a rust-lang/rust PR and\nnominate for lang team."]
+    ChangeAccepted["PR is merged."]
+    TypesTeamApproval["Types team approves the\ninteraction with type system."]
+    StyleTeamNotified["Style team notified about the new feature."]
+    
+    ExperimentalFeatureGateProposed --Seconded--> ExperimentalFeatureGateAccepted
+    ExperimentalFeatureGateAccepted --> RFCOpen
+    RFCOpen --FCP--> RFCAccepted
+    RFCAccepted --> FeatureComplete
+    FeatureComplete --> StabilizationProposed
+    %% StabilizationProposed --> StabilizationAccepted
+    StabilizationProposed --> Documented
+    Documented --> StabilizationAccepted
+    StabilizationProposed --> TypesTeamApproval
+    TypesTeamApproval --> StabilizationAccepted
+    StabilizationProposed --> StyleTeamNotified
+    StyleTeamNotified --> StabilizationAccepted
+    StabilizationAccepted -.if a change is needed.-> ChangeProposed
+    ChangeProposed --FCP proposed and accepted--> ChangeAccepted
+    ChangeAccepted -.if more changes are needed.-> ChangeProposed
+end
+
+TweakToExistingFeature --> ChangeProposed
+
+NewFeatureOrComplexChange --> RFCOpen
+
+LangTeamChampion -- Yes --> ExperimentalFeatureGateProposed
+
+LangTeamChampion -- No --> RFCOpen
+
+click RFCOpen href "https://github.com/rust-lang/rfcs/#when-you-need-to-follow-this-process" "Read about RFCs"
+click RFCAccepted href "https://forge.rust-lang.org/lang/rfc-merge-procedure.html" "RFC merge procedure"
+click ExperimentalFeatureGateProposed href "./experiment.html" "Read about experimental feature gates"
+click ExperimentalFeatureGateAccepted href "./experiment.html" "Read about experimental feature gates"
+click StabilizationProposed href "./stabilize.html" "Read about stabilization procedure"
+click StabilizationAccepted href "./stabilize.html" "Read about stabilization procedure"
+click Documented href "./stabilize.html" "Read about stabilization procedure"
+click TypesTeamApproval href "./stabilize.html" "Read about stabilization procedure"
+click StyleTeamNotified href "./stabilize.html" "Read about stabilization procedure"
```

Comment:
I think we discussed in-meeting the three arrows needing an "AND" of some kind for clarity.